### DISCUSSION TITLE
[Announcement] Final exam

### DISCUSSION THREAD

**Discussion by KihongHeo** [prosyslab-classroom/cs524-program-analysis#400] [Announcements]
- Time: Dec 13, 09:00 -- 11:00
- Place: Zoom
- Notes:
    - This will be an open-book exam. However, internet searching is not allowed.
    - Prepare a camera.
    - The problem set will be provided via screen share.
    - Students will take photos of their answer sheets and submit via Gradescope. We will open a test submission page later this week.
    - Write your answer in Korean unless you are more comfortable with English.

- Cheating will be strictly handled by the instructor as well as the department.
    - Do not have a discussion with others during the exam.
    - Do not distribute the problems and solutions.
    - Do not distribute all the homework test cases.

If you have more questions or suggestions, please write below.

**Comment by KihongHeo** [comment]
It will be awesome if you can scan your answer sheets using apps like the followings (if possible)
- iPhone: https://support.apple.com/ko-kr/HT210336
- Samsung: https://dorudoru.tistory.com/1645

**Comment by kangwoosukeq** [comment]
https://help.gradescope.com/article/0chl25eed3-student-scan-mobile-device

Here is a helpful link for you to follow.

We recommend you test your submission on Gradescope.

**Comment by jh05013** [comment]
Am I allowed to look at the lecture slides or my notes on my computer?

**Comment by KihongHeo** [comment]
It is not allowed to use phones or computers. Please print out all the things you want.

**Comment by KAIST-JongchanPark** [comment]
In the instruction, we need to prepare camera. Is it enough to have only labtop camera?

**Comment by KihongHeo** [comment]
Yes.
